README.md

```diff
@@ -2,7 +2,32 @@
 
 A gem to document components in GOV.UK frontend applications.
 
-## Usage
+![Screenshot of component guide](docs/screenshot.png)
+
+## Write components in your application
+
+Components are packages of template, style, behaviour and documentation that live in your application.
```

Comment:
This is nit-picking, I know, but can we really describe a set of files in different directories as a package? Perhaps an 'associated group of files'?